Sources/EdgeNetworkHandlers/EdgeConsentUpdate.swift

```diff
@@ -0,0 +1,24 @@
+//
+// Copyright 2021 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+import AEPServices
+import Foundation
+
+/// A request for sending a consent update to the Adobe Experience Edge.
+/// An `EdgeConsentUpdate` is the top-level request object sent to Experience Edge to the set-consent endpoint.
+struct EdgeConsentUpdate: Encodable {
+    /// The IdentityMap at the moment of this request
+    let identityMap: IdentityMap?
```

Comment:
nit: I think this is okay for now, in this PR, however when Edge reads the IdentityEdge XDM shared state, we should remove the IdentityMap class from Edge and use a `[String: Any]` type here.

Reply:
👍  should be handled by https://github.com/adobe/aepsdk-edge-ios/pull/165